app/assets/javascripts/discourse/app/components/welcome-topic-banner.js

```diff
@@ -0,0 +1,35 @@
+import { getOwner } from "discourse-common/lib/get-owner";
+import { action } from "@ember/object";
+import Component from "@ember/component";
+import Topic from "discourse/models/topic";
+import Composer from "discourse/models/composer";
+import { on } from "discourse-common/utils/decorators";
+
+export default Component.extend({
+  @on("didInsertElement")
```

Comment:
FYI, Glimmer components don't have the `didInsertElement` hook (although it can be added via render modifiers). 

I also find this a bit fragile, hiding the topic row via JS. Not sure absolutely have to hide it, tbh.

Reply:
Okay, I've removed the logic to hide welcome topic.